src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -158,11 +163,13 @@ object DemuxFastqs {
           batch
             .parWith(pool=pool)
             .map { readRecords => demultiplexer.demultiplex(readRecords: _*) }
+            .map { result => result.maskLowQualityBases(threshold=thresh, qualityEncoding=qualityEncoding) }
```

Comment:
what do you think the benefits of this are?
```suggestion
            .map { readRecords => 
                demultiplexer.demultiplex(readRecords: _*)
                    .maskLowQualityBases(threshold=thresh, qualityEncoding=qualityEncoding) 
            }
```